FRC1710-2019/src/main/java/frc/robot/intakePOS.java

```diff
@@ -0,0 +1,91 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+package frc.robot;
+
```

Comment:
Why are there two `package frc.robot;` inserts? Maybe remove the extra one?

Reply:
Good catch.